cmd/node/config/gasSchedules/gasScheduleV2.toml

```diff
@@ -0,0 +1,607 @@
+[BuiltInCost]
```

Comment:
Synthesis of the changes:
1. AoTPreparePerByte increased 4x
2. Elrond API costs increased some operations 10x-50x
3. Eth API costs increased some operations 10x-1000x
4. Big int API costs increased all operations 10x-60x except load & store
5. Crypto API cost increased the VerifyEd25519 operation 200x